### PR TITLE
LINQpad: Add previous version 5

### PR DESCRIPTION
**Pull request by Jogai**
No binaries added to the path to prevent a clash with linqpad 6.

### CODE REVIEW

**Comment by Jogai** [comment]
If anyone can help with the regex escape?

File: bucket/linqpad5.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    "hash": "7178cb0165963841fffbd0fce6a3f8d6c00571cb99622f1e5e472d523b91dd8c",
+    "shortcuts": [
+        [
+            "linqpad.exe",
+            "LINQPad"
+        ]
+    ],
+    "checkver": {
+        "url": "https://www.linqpad.net/Download.aspx",
+        "regex": "LINQPad5Version\\"\\>([\\d.]+)\\<"
```
Comment (issaclin32) [inline bucket/linqpad5.json:19]: should be `"regex": "LINQPad5Version\\\"\\>([\\d.]+)\\<"`
`\\` `\"` in json -> `\"` when loaded into the program

Comment (Jogai) [reply bucket/linqpad5.json:19]: Oh, it seems I have deleted my fork and the latest version is now 5.44.02. Maybe you can just copy the proposed changes with some edits (I dont car that its not my commit). Or should I make a new PR?

Comment (issaclin32) [reply bucket/linqpad5.json:19]: Just make a new PR.

**Pull request closed without merging**